pkgs/applications/networking/netperf/default.nix

```diff
@@ -1,21 +1,29 @@
-{ stdenv, fetchFromGitHub }:
+{ glibc, libsmbios, stdenv, autoreconfHook, fetchFromGitHub }:
 
 stdenv.mkDerivation rec {
-  name = "netperf-2.7.0";
+  name = "netperf-git";
```

Comment:
Please make it date with the day of commit instead (ex. 2018-05-09).

Reply:
will do, I meant to write a note if I should do that since I saw that in other nix files but they looked autogenerated so I wasn't sure.

Reply:
done